libs/SalesforceReact/SalesforceReactTests/ReactOauthTests.m

```diff
@@ -23,13 +22,24 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <UIKit/UIKit.h>
+#import "ReactTestCase.h"
 
+@interface ReactOauthTests : ReactTestCase
```

Comment:
Test for the oauth react bridge.
Only testing getAuthCredentials - didn't want to deal with logout since it brings up login and destroys the test_credentials refresh token.